src/main/java/gov/usgs/wma/waterdata/timeseries/TimeSeriesController.java

```diff
@@ -58,17 +63,31 @@ public String getStatisticalTimeSeries(@PathVariable(value = PARAM_COLLECTION_ID
 		return rtn;
 	}
 
-	@Operation(description = "Return GeoJSON Data specific to the requested Monitoring Location and Time Series.", responses = {
+	@Operation(description = "Return GeoJSON or Water ML 2 Data specific to the requested Monitoring Location and Time Series.", responses = {
 			@ApiResponse(responseCode = "200", description = "GeoJSON representation of the Time Series.", content = @Content(schema = @Schema(implementation = TimeSeriesGeoJSON.class))),
+			@ApiResponse(responseCode = "400", description = HTTP_400_DESCRIPTION, content = @Content(schema = @Schema(implementation = OgcException.class))),
 			@ApiResponse(responseCode = "404", description = HTTP_404_DESCRIPTION, content = @Content(schema = @Schema(implementation = OgcException.class))),
 			@ApiResponse(responseCode = "500", description = HTTP_500_DESCRIPTION, content = @Content(schema = @Schema(implementation = OgcException.class))) }, externalDocs = @ExternalDocumentation(url = "https://github.com/opengeospatial/omsf-profile/tree/master/omsf-json"))
-	@GetMapping(value = URL_STATISTICAL_TIME_SERIES, produces = MediaType.APPLICATION_JSON_VALUE)
+	@GetMapping(value = URL_STATISTICAL_TIME_SERIES, produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE})
 	public String getTimeSeries(@PathVariable(value = PARAM_COLLECTION_ID) String collectionId, // ex: networkId,
 			@PathVariable(value = PARAM_FEATURE_ID) String featureId,
 			@PathVariable(value = PARAM_TIME_SERIES_ID) String timeSeriesId, // ex: 2ae58a8bdb1b4b778577a2ce3a5362d0
-			HttpServletResponse response) {
+			@Parameter(in = ParameterIn.QUERY, description = contentTypeDesc, schema = @Schema(type = "string"), examples = {
+					@ExampleObject(name = "json", value = "json", description = "GeoJSON"),
+					@ExampleObject(name = "xml", value = "xml", description = "Water ML 2")})
+			@RequestParam(value = "f", required = false, defaultValue = "json")
+			String mimeType,
+			HttpServletResponse response)
+					throws HttpMediaTypeNotAcceptableException {
+
+		determineContentType(mimeType);
+		String rtn = null;
+		if (contentIsJson()) {
+			rtn = timeSeriesDao.getTimeSeries(collectionId, featureId, timeSeriesId);
+		} else if (contentIsXml()) {
+			rtn = timeSeriesDao.getTimeSeriesWaterML(collectionId, featureId, timeSeriesId);
+		}
```

Comment:
This will ignore the http accept header - it seems like we also want to consider that.
Also, don't we want an else block that would throw a 415 - Unsupported Media Type.
And where do we actually set the return type in the response?

Reply:
I was leaving the accept header to another ticket. The current content query param (f) defaults to json. In the content
negotiation  scheme as in WQP, the parameter is preferred.
Leaving the parameter with a default would essentially mean the accept header is never used. Having no default value has wider implications as far as compliance to the OGC standard, etc.

Good catch on the content Type.